.editorconfig

```diff
@@ -1,15 +0,0 @@
-# editorconfig.org
```

Comment:
Are we deleting this because we don't find any value in it?

Reply:
Was just temporary, because it was messing up some formatting for me